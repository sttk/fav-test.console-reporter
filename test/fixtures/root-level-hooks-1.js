'use strict';

beforeEach(function() {
  console.log('before every test in every file');
});

it('test', function() {
  console.log('test run');
});

describe('suite1', function() {
  it('test1', function() {
    console.log('test1 run');
  });

  describe('suite2', function() {
    it('test2', function() {
      console.log('test2 run');
    });
  });
});
