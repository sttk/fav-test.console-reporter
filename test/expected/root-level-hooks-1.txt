

before every test in every file
test run
  ✓ test

  suite1
before every test in every file
test1 run
    ✓ test1
    suite2
before every test in every file
test2 run
      ✓ test2


  3 passing (4ms)

